README.rst

```diff
@@ -158,12 +27,14 @@ Reporting Security Issues
 
 Please do not report security issues in public. Please email security@edx.org
 
-Mailing List and IRC Channel
-============================
+Mailing List and Slack
+======================
+
+You can discuss this code on the `edx-code Google Group`. Alternatively, our real-time conversations are on Slack. You can request a `Slack invitation`_, then join our `community Slack team`_.
 
-You can discuss this code on the
-`edx-code Google Group <https://groups.google.com/forum/#!forum/edx-code>`_ or
-in the `edx-code` IRC channel on Freenode.
+.. _edx-code Google Group: https://groups.google.com/forum/#!forum/edx-code
+.. _Slack invitation: https://openedx-slack-invite.herokuapp.com/
+.. _community Slack team: http://openedx.slack.com/
 
```

Comment:
Although this is a copy of a change I made elsewhere, these days I think it would be better to point to open.edx.org/getting-help.  Let's do this:
```
You can get help with this code on our `mailing lists`_ or in real-time conversations on `Slack`_.

.. _mailing lists: https://open.edx.org/getting-help
.. _Slack: https://open.edx.org/getting-help
```
(Yes, it's a little weird to link them both to the same place, but I think it's a good compromise between having a content-less link to the page, and too many details here.)